Apsitvarkom.UnitTests/Models/Validation/PollutedLocationDTOValidationTests.cs

```diff
@@ -36,8 +36,8 @@ public class PollutedLocationDTOValidationTests
             Notes = "",
             Progress = 0,
             Radius = 0,
-            Severity = "Whatever",
-            Spotted = "timestamp"
+            Severity = "Low",
+            Spotted = "1889-04-20"
         },
     };
 
```

Comment:
Fix data for old InvalidInputDTOs, as when some properties didn't necessarily have to be validated before, the DTO on line 65 no longer breaks due to missing Location as expressed in the comments, but before that in the Id check and etc.

Reply:
Github for some reason didn't allow me to select the proper lines as no changes were done there, so I've mentioned the line in the comment.

Reply:
Great point.